apps/meteor/tests/end-to-end/api/04-direct-message.js

```diff
@@ -355,63 +377,55 @@ describe('[Direct Messages]', function () {
 	});
 
 	describe('/im.messages.others', () => {
```

Comment:
Can you add a case for `endpoint disabled but user with permission`?

Reply:
I believe this is what the `should fail when the endpoint is disabled` does already :thinking: 
I'll update its title to make this clear